docs/public/using/backend/rpcapi-reference.md

```diff
@@ -61,8 +66,10 @@ This API is implemented using [JSON-RPC 2.0].
 JSON-RPC request objects are accepted in the body of HTTP POST requests to any path.
 The HTTP request must contain the header `Content-Type: application/json`.
 
-All JSON-RPC request and response objects have the keys `"jsonrpc"`, `"id"` and `"method"`.
-For details on these, refer to the [JSON-RPC 2.0] specification.
+All JSON-RPC request objects have the keys `"jsonrpc"`, `"id"` and `"method"`.
+The response object have the same keys `"jsonrpc"` and `"id"`. For details on
+these, refer to the [JSON-RPC 2.0] specification. The `"jsonrpc"` and `"id"` keys
+are assumed in the specification of the methods below.
```

Comment:
Do you know what purpose this paragraph serves? AFAICT it only restates things from the specification. It's better to just refer to the specification. IMHO this entire paragraph could be removed.